apps/example/src/App.jsx

```diff
@@ -59,12 +66,19 @@ const App = (props = {}) => {
       <MonorepoChecker></MonorepoChecker>
       <AsyncWorker consumerId={props.id} mockAPI={true} />
       <AppShell
+        fullWidthContent
+        embedded={props.embedded === "true" || props.embedded === true}
```

Comment:
**(Question)**

Wondering why we check for `string` value here if `embedded` has `PropTypes.bool `?
We'll be refactoring this anyway, so can leave it. But just curious. 🙈

Reply:
Unfortunately, I can't answer that either

Reply:
The Example App was previously integrated into web pages via a <script> tag using a widget loader, which only allowed string-based props to be passed. However, we have since moved away from using the widget loader and now utilize direct dynamic imports. As a result, the reliance on string-based props is a legacy from the previous approach.